ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -123,6 +140,30 @@ class UnconnectedTemplatesIndex extends Component<Props> {
     )
   }
 
+  private applyPackages = async (orgID, templateName) => {
+    const yamlLocation =
+      getRawYamlFromGithub(this.state.currentTemplate) + `/${templateName}.yml`
+
+    const params = {
+      data: {
+        dryRun: true,
+        orgID,
+        remotes: [{url: yamlLocation}],
+      },
+    }
+    try {
+      const resp = await postPackagesApply(params)
+      if (resp.status < 300) {
+        const template = resp.data.summary
+        this.props.setActiveCommunityTemplate(template)
+        return template
+      }
+      console.error('Failed to apply packages: ', resp)
```

Comment:
i've seen the pattern elsewheres in our codebase to flip the conditional logic and throw an error so that we can address how we handle an error (maybe changing from console errors to honey badger alerts) in the same place (the catch)

ie:
```
if (resp.status >= 300) {
  throw new Error(..same mesage..)
}
// you got it
```

Reply:
sure, works for me